### PR TITLE
register Portland State Hosted CE

### PR DESCRIPTION
**Pull request by jdost321**
I'm not sure I registered their squid correctly, just did it on @brianhlin 's request here:

https://opensciencegrid.freshdesk.com/a/tickets/69934

### CODE REVIEW

**Review by brianhlin — APPROVED** [review]
LGTM but I don't see a Squid registration?

**Comment by jdost321** [comment]
sorry missed the commit.. just added it

File: topology/Portland State University/PDX-OIT/PDX-Coeus.yaml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          Name: Jeffrey Michael Dost
+          ID: 3a8eb6436a8b78ca50f7e93bb2a4d1f0141212ba
+      Security Contact:
+        Primary:
+          Name: Jeffrey Michael Dost
+          ID: 3a8eb6436a8b78ca50f7e93bb2a4d1f0141212ba
```
Comment (brianhlin) [inline topology/Portland State University/PDX-OIT/PDX-Coeus.yaml:149]:
```suggestion
        Primary:
          Name: Marko Markoc
          ID: OSG1000189
        Secondary:
          Name: Gary Sandine
          ID: OSG1000190
        Tertiary:
          Name: Jim Stapleton
          ID: OSG1000191
      Security Contact:
        Primary:
          Name: Marko Markoc
          ID: OSG1000189
        Secondary:
          Name: Gary Sandine
          ID: OSG1000190
        Tertiary:
          Name: Jim Stapleton
          ID: OSG1000191
```

The site admins are responsible for the Squid service

File: topology/Portland State University/PDX-OIT/PDX-Coeus.yaml [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        #   ID: <EMAIL HASH>
+        # Tertiary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+
+      # Site contact (optional) are persons or groups of people (i.e.,
+      # mailing lists) that are generally responsible for a site's
+      # relationship with the OSG (e.g., principal investigators,
+      # local administrator contact for OSG Hosted CEs)
+      # Site Contact:
```
Comment (brianhlin) [inline topology/Portland State University/PDX-OIT/PDX-Coeus.yaml:66]:
```suggestion
      Site Contact:
        Primary:
          Name: Marko Markoc
          ID: OSG1000189
        Secondary:
          Name: Gary Sandine
          ID: OSG1000190
        Tertiary:
          Name: Jim Stapleton
          ID: OSG1000191
```

**Review by brianhlin — APPROVED** [review]
Some updates to the site contacts, otherwise LGTM

**Pull request merged**